doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -87,37 +87,6 @@ In order for the Kubernetes components to be conformant with the Reference Archi
 
 <p align="center"><b>Table 4-2:</b> Kubernetes Specifications</p>
 
-<!--
-> THE BELOW TEXT HAS BEEN COMMENTED AS NEEDS REVIEWING AND REPLACED WITH SPECS IN THE ABOVE TABLE AS PER:
-#1635
-
-
-
-This Reference Architecture also specifies:
-
-- Master nodes must run the following Kubernetes control plane services:
-    - kube-apiserver
-    - kube-scheduler
-    - kube-controller-manager
-- Master nodes can also run the etcd service and host the etcd database, however etcd can also be hosted on separate nodes if desired
```

Comment:
Where is this captured in the requirements?

Reply:
@pgoyal01 , @EditKoselak  I will crosscheck these with the requirements and:
  - Create an issue and pr if something is missing
  - Comment in this pr once all relevant issues are colsed

Reply:
#2283 is to cover the worker node part.

Reply:
#2284 is to cover the workload part.

Reply:
kubelet part is covered by `ra2.k8s.006`.

Reply:
etcd bullet still missing